WordPressVIPMinimum/Tests/VIP/AdminBarRemovalUnitTest.inc

```diff
@@ -0,0 +1,108 @@
+<?php
+
+add_filter( 'show_admin_bar', '__return_false' ); // Bad.
+add_filter( 'show_admin_bar', '__return_true' ); // Ok.
+
+show_admin_bar( false ); // Bad.
+show_admin_bar( true ); // Ok.
+
+add_filter( 'show_admin_bar', 'my_own_return_false' ); // Bad.
+
+// @codingStandardsChangeSetting WordPressVIPMinimum.VIP.AdminBarRemoval remove_only false
```

Comment:
These annotations will need updating - see https://github.com/WordPress-Coding-Standards/WordPress-Coding-Standards/pull/1571

All of the .inc files in this PR will need checking and fixing up please.